File: Assignment_27/program4.c
//////////////////////////////////////////////////////////////////////////////
//
// Name :          program4.c
// Discription :   Copy small characters of that string into another string
// Input :         String
// Output :        String
// Author :        Shivam Rajendra Kale
// Date :          10/06/2025
//
//////////////////////////////////////////////////////////////////////////////

/*
   Start
      Accept string from user 
           Copy small characters of that string into another string   
    Stop         
*/

#include<stdio.h>

void StrCpyCap(char *str, char *dest)
{
   if(NULL == str)
   {
      return ;
   }

   while(*str != '\0')
   {
      if((*str >= 'a') && (*str <= 'z') || (*str == ' '))
      {
         *dest = *str;
         dest++;
      }
      str++;
   }

}
int main()
{
   char arr[30] = {'\0'};
   char brr[30] = {'\0'};

   printf("Enter String :\n");
   scanf("%[^'\n']s",arr);

   StrCpyCap(arr,brr);

   printf("%s",brr);

   return 0;
}

File: Assignment_52/program4.java
//////////////////////////////////////////////////////////////////////////
//
// Name :          program4.java
// Discription :   use to Combination of two Array
// Input :         Integer
// Output :        Integer
// Author :        Shivam Rajendra Kale
// Date :          16/10/2025
//
//////////////////////////////////////////////////////////////////////////

import java.util.*;

class MyArray
{
    public int[] ArrayConcate(int Arr[], int Brr[])
    {
        int i = 0, j = 0;

        int Crr[] = new int[Arr.length + Brr.length];

        for(i = 0; i < Arr.length; i++)
        {
            Crr[i] = Arr[i];  
        }

        for(i = 0, j = Arr.length; i < Brr.length; i++, j++)
        {
            Crr[j] = Brr[i];         
        } 

        return Crr;
    }
}

class program4
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter isze of first Array : ");
        int iSize = sobj.nextInt();

        int Arr[] = new int [iSize];

        System.out.println("Enter elements : ");
        for(int i = 0; i < iSize; i++)
        {
            Arr[i] = sobj.nextInt();
        }

        System.out.println("Enter isze of second Array : ");
        int iSize1 = sobj.nextInt();

        int Brr[] = new int [iSize1];

        System.out.println("Enter elements : ");
        for(int i = 0; i < iSize1; i++)
        {
            Brr[i] = sobj.nextInt();
        }

        MyArray mobj = new MyArray();

        int Crr[] = mobj.ArrayConcate(Arr, Brr);

        for(int i = 0; i < Crr.length; i++)
        {
            System.out.print(Crr[i]+"\t");
        }

        sobj.close();
    }
}